infra/travis/travis_build.py

```diff
@@ -97,6 +98,11 @@ def build_project(project):
     build_fuzzers(project, sanitizer, 'libfuzzer')
     check_build(project, sanitizer, 'libfuzzer')
 
+  if 'i386' in project_yaml.get('architectures', ['x86_64']):
```

Comment:
nit: add a DEFAULT_ARCHITECTURES for consistency.

Reply:
I went with using an empty list since I don't need the default list.